README.md

```diff
@@ -1,11 +1,16 @@
 
-# insights-frontend-starter-app
+# Sources
+
+List of sources miniapp for Insights.
+
+:![Main screen with "Add a new Source" modal](https://raw.githubusercontent.com/martinpovolny/topological_inventory-ui/master/doc/images/sources-main-add.jpg)
```

Comment:
This stays in preview
![screenshot-github com-2019 02 27-09-33-41](https://user-images.githubusercontent.com/22619452/53478089-564dc500-3a76-11e9-9c99-a07951332ffd.png)